core/src/main/scala/dagr/core/config/Configuration.scala

```diff
@@ -104,124 +51,19 @@ private[core] object Configuration extends ConfigurationLike {
   * that is then mixed into the Configuration object above, and used to create a concrete
   * Configuration trait below.
   */
-private[config] trait ConfigurationLike extends LazyLogging {
-  private[config] def config : Config
+private[config] trait ConfigurationLike extends LazyLogging with dagr.sopt.util.ConfigurationLike {
 
-  /**
-    * Looks up a single value of a specific type in configuration. If the configuration key
-    * does not exist, an exception is thrown. If the requested type is not supported, an
-    * exception is thrown.
-    */
-  def configure[T : TypeTag](path: String) : T = {
-    Configuration.RequestedKeys += path
+  // route error messages to our logger
+  override protected def error(parts: Any*) : Unit = logger.error(parts:_*)
```

Comment:
When `Configuration` was just a class in `dagr` I think having it log errors made sense.  But if it's getting extracted somewhere more common (`sopt` or `commons`) then I don't think it's really appropriate for it to be logging - it should just throw exceptions with well-formatted errors.